### ISSUE TITLE
Protector Block

### ISSUE THREAD

**Issue by IntegratedQuantum** [PixelGuys/Cubyz#81]
The protector block protects a chunk from modifications (including e.g. chest inventories) by another player.
Protector blocks can be shared between players, to allow team play.
Additionally a weaker version(that allows placing some blocks like torches) of the protector block can be owned by bosses. This allows protecting a dungeon until the player defeats the final boss.

**Comment by archbirdplus** [comment]
Protector blocks simply denying modifications feels artificial. Perhaps they would instead spawn bosses upon modification. Defeating the boss might then disable the protector, or allow a single modification, or kill the owning player(s) who are linked to it. Connecting the power of the protector block with the life force of its owner feels like something worth exploring.

**Comment by IntegratedQuantum** [comment]
Yeah, that last idea seems good, since the player has to be online to die.

**Comment by Argmaster** [comment]
I also suggested in discord that there should be force field block allowing owner of the block to deny entrance into claimed region for all unwanted intruders. Maybe they would stack with respawn block into a totem? This would set a theme for them.
